pkg/storage/bloom/v1/bloom.go

```diff
@@ -167,6 +166,13 @@ func (d *BloomPageDecoder) Err() error {
 	return d.err
 }
 
+func (d *BloomPageDecoder) Close() {
+	d.err = nil
+	d.cur = nil
+	d.dec.B = d.data[:0]
```

Comment:
why not setting it to nil ? this is scary to keep a ref that is being put back to the pool.

Reply:
Right, it can be set to `nil`.